jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -176,20 +208,19 @@ local patchOrExcludeRuleGroup(group, groupSet, operation) =
   // 'alert' or 'record' identifier. The function will apply the patch to every
   // alerting/recording rule in the group whose name starts by the identifier.
   patchRules(o): {
-    local patch(o) = o {
-      [if (o.kind == 'PrometheusRule') then 'spec']+: {
-        groups: std.map(
-          function(group)
-            group {
-              rules: std.flattenArrays(
-                patchOrExcludeRuleGroup(group, patchedRules, 'patch')
-              ),
-            },
-          super.groups,
-        ),
-      },
-    },
-    [k]: patch(o[k])
+    [k]: $.patchRule(o[k])
+    for k in std.objectFields(o)
+  },
+
+  // removeRunbookUrl removes runbook_url from alert annotations, but skips alerts that are in includeRunbooks list
+  removeRunbookUrl(o): {
+    [k]: $.removeRunbookAnnotation(o[k])
+    for k in std.objectFields(o)
+  },
+
+  // shorthand for rule patching, rule excluding, and runbook_url removal
+  sanitizeAlertRules(o): {
+    [k]: $.removeRunbookAnnotation($.patchRule($.excludeRule(o[k])))
```

Comment:
Shouldn't this call `removeRunbookUrl` instead of `removeRunbookAnnotation`?

Reply:
`removeRunbookUrl` was already taken in https://github.com/openshift/cluster-monitoring-operator/pull/1315/files#diff-9425bc6534201e7b9f2c51476ae71ce5e7c9a353c12c58ec44008c7f12d0171fR131

Reply:
I got confused by https://github.com/openshift/cluster-monitoring-operator/pull/1315/files#diff-9425bc6534201e7b9f2c51476ae71ce5e7c9a353c12c58ec44008c7f12d0171fR215-R219, do you know if this is really needed?

Reply:
Ah, you're right, this is confusing and unused. I removed it in latest commit.

Reply:
Thanks!